CenterForOpenScience/ember-osf-web
# [ENG-4020] Deprecation/mutation after consumption

-   Ticket: [ENG-4020]
-   Feature flag: n/a

## Purpose

Un-silence the deprecation for mutation after consumption and fix the problems that arise. Basically, things can't happen on init or in construction if they might write to a variable being read on render, so this puts those times it happened into a bit later of a part of the process.

## Summary of Changes

1. Un-silence the deprecation
2. Fix File Browser breadcrumbs and upload widget
3. Fix make-decision-dropdown
4. Fix moderators list row

## Side Effects

Possibly.

## QA Notes

It's possible I broke the limiter for the number of uploads per provider, so that's the main thing to check. 


[ENG-4020]: https://openscience.atlassian.net/browse/ENG-4020?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
